ComponentKitTests/TransactionalDataSource/CKTransactionalComponentDataSourceTests.mm

```diff
@@ -192,6 +196,34 @@ - (void)testSynchronousReloadCancelsPreviousAsynchronousReload
   }));
 }
 
+- (void)testAsynchronouslyInsertingItemsAppliesModificationsOnWorkThread
```

Comment:
My usual strategy is to not test threading-related things. Too flaky and nondeterminstic.